Marksman/Toc.fs

```diff
@@ -27,15 +27,19 @@ module Entry =
     let Mk (level: EntryLevel, title: Title) =
         { level = level; title = title; link = Slug.ofString title }
 
-    let renderLink entry =
-        let offset = String.replicate (entry.level - 1) " "
+    let renderLink entry minLevel =
+        let offset = String.replicate (entry.level - minLevel) "  "
```

Comment:
If all you have is headings from 2 to 5, no need to render them shifted by too many spaces to the right